src/test/java/org/jpeek/MainTest.java

```diff
@@ -67,6 +69,47 @@ public void crashesIfNoOverwriteAndTargetExists() throws IOException {
         );
     }
 
+    @Test
+    public void crashesIfOverwriteAndSourceEqualsToTarget() throws IOException {
+        final Path source = Files.createTempDirectory("sourceequalstarget");
+        final Path target = source;
+        new Assertion<>(
+            "Must throw an exception",
+            () -> {
+                Main.main(
+                    "--sources", source.toString(),
+                    "--target", target.toString(),
+                    "--overwrite"
+                );
+                return null;
```

Comment:
@humb1t Let's remove this null from here, I know that this method won't ever be called but it is still a bad pratice.

Reply:
I return true now